Bugsnag/include/Bugsnag/Bugsnag.h

```diff
@@ -140,6 +141,18 @@ BUGSNAG_EXTERN
  */
 + (void)notify:(NSException *_Nonnull)exception;
 
+/**
+ * Send a custom or caught exception to Bugsnag.
+ *
+ * The exception will be sent to Bugsnag in the background allowing your
+ * app to continue running.
+ *
+ * @param exception  The exception.
+ * @param options Defines the set of granular flags for controlling data capture at `notify` time.
```

Comment:
nit: the spacing of these comments isn't consistent

```suggestion
 * @param exception The exception.
 * @param options Defines the set of granular flags for controlling data capture at `notify` time.
```